cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,314 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"github.com/awslabs/aws-sdk-go/aws"
```

Comment:
Before merging these will need to be vendored using `party`. @mjibson can help you with this.